.npmignore

```diff
@@ -1,15 +1,19 @@
+/.docz
```

Comment:
@bedrich-schindler Could you please check this file for me? You know better what should be here and what not… Thanks!

Reply:
I do not have latest version of documentation running up, so if you include all generated files and files connected with documentation, it should be ok than.

I do not know if we want to include MDX files in npm package as well. It might be useful, but I do not know if it is correct use case. So it is up to you.